src/jquery.multiselect.js

```diff
@@ -510,17 +654,24 @@
         .find('span')
         .on('click.multiselect,click', clickHandler);
     },
+
     /**
-     * Bind events to the menu for options and option groups
-     * This methond scopes actions to filtered options
+     * Bind events to the checkboxes for options and option groups
+     * Must be bound to the checkboxes container.
+     * This method scopes actions to filtered options
      * Called by _bindEvents
      */
     _bindMenuEvents: function() {
       var self = this;
+
       // optgroup label toggle support
-      this.$menu.on('click.multiselect', '.ui-multiselect-optgroup a', function(e) {
+      self.$checkboxes.on('click.multiselect', 'a', function(e) {
```

Comment:
If this is now binding to checkboxes instead of menu, the method name should change accordingly.